query/src/sql/planner/binder/ddl/account.rs

```diff
@@ -0,0 +1,210 @@
+            }
+            GrantSource::ALL { level } => {
+                // ALL PRIVILEGES have different available privileges set on different grant objects
+                // Now in this case all is always true.
+                let grant_object = self.convert_to_grant_object(level);
+                let priv_types = grant_object.available_privileges();
+                let plan = GrantPrivilegePlan {
+                    principal: principal.clone(),
+                    on: grant_object,
+                    priv_types,
+                };
+                Ok(Plan::GrantPriv(Box::new(plan)))
+            }
+            GrantSource::Privs { privileges, level } => {
+                let grant_object = self.convert_to_grant_object(level);
+                let mut priv_types = UserPrivilegeSet::empty();
+                for x in privileges {
+                    priv_types.set_privilege(*x);
+                }
+                let plan = GrantPrivilegePlan {
+                    principal: principal.clone(),
+                    on: grant_object,
+                    priv_types,
+                };
+                Ok(Plan::GrantPriv(Box::new(plan)))
+            }
+        }
+    }
+
+    pub(in crate::sql::planner::binder) async fn bind_revoke(
+        &mut self,
+        stmt: &GrantStatement,
+    ) -> Result<Plan> {
+        let GrantStatement { source, principal } = stmt;
+
+        match source {
+            GrantSource::Role { role } => {
+                let plan = RevokeRolePlan {
+                    principal: principal.clone(),
+                    role: role.clone(),
+                };
+                Ok(Plan::RevokeRole(Box::new(plan)))
+            }
+            GrantSource::ALL { level } => {
+                // ALL PRIVILEGES have different available privileges set on different grant objects
+                // Now in this case all is always true.
+                let grant_object = self.convert_to_grant_object(level);
+                let priv_types = grant_object.available_privileges();
+                let plan = RevokePrivilegePlan {
+                    principal: principal.clone(),
+                    on: grant_object,
+                    priv_types,
+                };
+                Ok(Plan::RevokePriv(Box::new(plan)))
+            }
+            GrantSource::Privs { privileges, level } => {
+                let grant_object = self.convert_to_grant_object(level);
+                let mut priv_types = UserPrivilegeSet::empty();
+                for x in privileges {
+                    priv_types.set_privilege(*x);
+                }
+                let plan = RevokePrivilegePlan {
+                    principal: principal.clone(),
+                    on: grant_object,
+                    priv_types,
+                };
+                Ok(Plan::RevokePriv(Box::new(plan)))
+            }
+        }
+    }
+
+    //Copy from query/src/sql/statements/statement_grant.rs
```

Comment:
statement_grant.rs is going to be deleted, so this comment will become outdated soon. Consider adding a more informative comment or removing it.

Reply:
Fine. I will delete it.